### ISSUE TITLE
Display information about the code repository

### ISSUE THREAD

**Issue by orviz** [EOSC-synergy/sqaaas-web#154]
The final view of the report shall include information relative to the code repository/ies being assessed, including:
- URL (with link) and description of each repository
- Release version: this shall be highlighted as it is the source of the quality assessment
- Any other metrics provided by the API (stars, etc.)

**Comment by orviz** [comment]
A tentative specification is available at sqaaas-api-spec's [feature/repo_data](https://github.com/EOSC-synergy/sqaaas-api-spec/tree/feature/repo_data) branch. It adds a new property `repository` with the data to be displayed:

```json
{
  "repository": [
    {
      "name": "sqaaas-api-spec",
      "url": "https://github.com/eosc-synergy/sqaaas-api-spec",
      "description": "OpenAPI specification for the SQA as a Service",
      "language": "Shell",
      "tag": "2.5.0",
      "topics": [
        "openapi",
        "sqaaas"
      ],
      "stargazers_count": 2,
      "watchers_count": 2,
      "contributors_count": 2,
      "forks_count": 2
    }
  ],
  "report": {
  },
  "badge": {
  }
}
```

**Comment by orviz** [comment]
Latest screenshot from David:

![image](https://user-images.githubusercontent.com/1225231/190093238-733642e1-978f-4a11-9f9f-d1a5610c7aa1.png)

**Comment by orviz** [comment]
Released under [1.5.0](https://github.com/EOSC-synergy/sqaaas-web/releases/tag/1.5.0)
